inspirehep/config.py

```diff
@@ -1408,173 +1410,11 @@
-    "math.GM": "Math and Math Physics",
-    "math.GN": "Math and Math Physics",
-    "math.GR": "Math and Math Physics",
-    "math.GT": "Math and Math Physics",
-    "math.HO": "Math and Math Physics",
-    "math.IT": "Math and Math Physics",
-    "math.KT": "Math and Math Physics",
-    "math.LO": "Math and Math Physics",
-    "math.MG": "Math and Math Physics",
-    "math.MP": "Math and Math Physics",
-    "math.NA": "Math and Math Physics",
-    "math.NT": "Math and Math Physics",
-    "math.OA": "Math and Math Physics",
-    "math.OC": "Math and Math Physics",
-    "math.PR": "Math and Math Physics",
-    "math.QA": "Math and Math Physics",
-    "math.RA": "Math and Math Physics",
-    "math.RT": "Math and Math Physics",
-    "math.SG": "Math and Math Physics",
-    "math.SP": "Math and Math Physics",
-    "math.ST": "Math and Math Physics",
-    "nlin": "General Physics",
-    "nlin.AO": "General Physics",
-    "nlin.CD": "General Physics",
-    "nlin.CG": "General Physics",
-    "nlin.PS": "Math and Math Physics",
-    "nlin.SI": "Math and Math Physics",
-    "nucl-ex": "Experiment-Nucl",
-    "nucl-th": "Theory-Nucl",
-    "patt-sol": "Math and Math Physics",
-    "physics": "General Physics",
-    "physics.acc-ph": "Accelerators",
-    "physics.ao-ph": "General Physics",
-    "physics.atm-clus": "General Physics",
-    "physics.atom-ph": "General Physics",
-    "physics.bio-ph": "Other",
-    "physics.chem-ph": "Other",
-    "physics.class-ph": "General Physics",
-    "physics.comp-ph": "Computing",
-    "physics.data-an": "Data Analysis and Statistics",
-    "physics.ed-ph": "Other",
-    "physics.flu-dyn": "General Physics",
-    "physics.gen-ph": "General Physics",
-    "physics.geo-ph": "General Physics",
-    "physics.hist-ph": "Other",
-    "physics.ins-det": "Instrumentation",
-    "physics.med-ph": "Other",
-    "physics.optics": "General Physics",
-    "physics.plasm-ph": "General Physics",
-    "physics.pop-ph": "Other",
-    "physics.soc-ph": "Other",
-    "physics.space-ph": "Astrophysics",
-    "q-alg": "Math and Math Physics",
-    "q-bio": "Other",
-    "q-bio.BM": "Other",
-    "q-bio.CB": "Other",
-    "q-bio.GN": "Other",
-    "q-bio.MN": "Other",
-    "q-bio.NC": "Other",
-    "q-bio.OT": "Other",
-    "q-bio.PE": "Other",
-    "q-bio.QM": "Other",
-    "q-bio.SC": "Other",
-    "q-bio.TO": "Other",
-    "q-fin": "Other",
-    "q-fin.CP": "Other",
-    "q-fin.EC": "Other",
-    "q-fin.GN": "Other",
-    "q-fin.MF": "Other",
-    "q-fin.PM": "Other",
-    "q-fin.PR": "Other",
-    "q-fin.RM": "Other",
-    "q-fin.ST": "Other",
-    "q-fin.TR": "Other",
-    "quant-ph": "General Physics",
-    "solv-int": "Math and Math Physics",
-    "stat": "Other",
-    "stat.AP": "Other",
-    "stat.CO": "Other",
-    "stat.ME": "Other",
-    "stat.ML": "Other",
-    "stat.OT": "Other",
-    "stat.TH": "Other"
-}
+ARXIV_TO_INSPIRE_CATEGORY_MAPPING = schema_utils.ARXIV_TO_INSPIRE_CATEGORY_MAPPING
```

Comment:
You don't even need this: importing the name creates the binding, so just doing `from inspire_schemas.utils import ARXIV_TO_INSPIRE_CATEGORY_MAPPING` is enough.